src/lambdas/mps-report/src/generateReport.ts

```diff
@@ -0,0 +1,139 @@
+import getCourtErrors from "./getCourtErrors"
+import { headers } from "../config/mpsHeadingKeys"
+import getOffenceCode from "./getOffenceCode"
+import AnnotatedHearingOutcome from "./types/AnnotatedHearingOutcome"
+import { PostgresGateway } from "@bichard/postgres-gateway"
+import { isError } from "@bichard/types"
+import { xml2js } from "xml-js"
+import { getText } from "./types/XmlStringCell"
+import getResultQualifierCodes from "./getResultQualifierCodes"
+import getDurationAndAmount from "./getDurationAndAmount"
+import OffenceDetails, { isMultiple } from "./types/OffenceDetails"
+import OffenceResult from "./types/OffenceResult"
+import HearingOutcomeCase from "./types/HearingOutcomeCase"
+
+interface ReportRowResultQuery {
+    court_date: string
+    court_code: string
+    force_code: string
+    is_urgent: string
+    defendant_name: string
+    triggers: string
+    error_report: string
+}
+
+function UpdateCommonHeaders(row: ReportRowResultQuery, hearingOutcomeCase: HearingOutcomeCase, offence:OffenceDetails) {
+    let newRow = []
```

Comment:
I wonder if it's cleaner to simply return an array here:

```
return [
  row.court_date,
  row.court_code,
  ...,
  getText(hearingOutcomeCase.HearingDefendant.DefendantDetail?.BirthDate),
  ...
]
```

Reply:
yup, sounds good actually, will do that